aws/logs_monitoring/tools/publish_prod.sh

```diff
@@ -2,47 +2,53 @@
 
 # Use with `./publish_prod.sh <DESIRED_NEW_VERSION>
 
-set -e
+set -o nounset -o pipefail -o errexit
 
-# Ensure on main, and pull the latest
-BRANCH=$(git rev-parse --abbrev-ref HEAD)
-if [ $BRANCH != "master" ]; then
-    echo "Not on master, aborting"
+log_info() {
+    local BLUE='\033[0;34m'
+    local RESET='\033[0m'
+
+    printf -- "%b%b%b\n" "${BLUE}" "${*}" "${RESET}" 1>&2
+}
+
+log_error() {
+    local RED='\033[0;31m'
+    local RESET='\033[0m'
+
+    printf -- "%b%b%b\n" "${RED}" "${*}" "${RESET}" 1>&2
     exit 1
-else
-    echo "Updating master"
-    git pull origin master
```

Comment:
I don't pull master anymore in the subscript, the main release script is already doing it.